webrtc/go/manifest.toml

```diff
@@ -0,0 +1,22 @@
+name = "webrtc:go"
+
+[builders]
+"exec:go"   = { enabled = true }
+
+[builders."docker:go"]
+enabled                = true
+build_base_image       = "golang:1.19-bullseye"
+
+[runners]
+"local:docker"  = { enabled = true }
+"local:exec"    = { enabled = true }
```

Comment:
why is `local:exec` and `cluster:k8s` enabled?

Reply:
Didn't think about it too much, but is there any particular reason this test couldn't be run in those circumstances?

Reply:
I would think that local:exec needs a different builder, the k8s one I do not know